examples/MQ/Lmd/runMBSSink.cxx

```diff
@@ -6,48 +6,82 @@
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
 
-#include "BaseMQFileSink.h"
-#include "FairMBSRawItem.h"   // data type for the OutputPolicy
+#include "FairMBSRawItem.h" // data type
+
+#include "FairMQ.h" // for fair::mq::Device
 #include "FairRunFairMQDevice.h"
-#include "IOPolicy.h"             // fair::mq::policy::
-#include "RootOutFileManager.h"   // OutputPolicy
-#include "RootSerializer.h"       // RootSerializer
+#include "RootSerializer.h" // RootSerializer
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TClonesArray.h>
 
-#include <TClonesArray.h>   // data type for the InputPolicy
+#include <fairlogger/Logger.h>
 
-// InputPolicy - initialize input and deserialize message into it
-using RootDefaultInputPolicy =
-    fair::mq::policy::InputPolicy<RootSerializer,                         // deserializer from msg to input
-                                  TClonesArray,                           // input data type
-                                  fair::mq::policy::PointerType,          // input pointer type (automatically selected)
-                                  fair::mq::policy::OpNewCreator,         // input allocation
-                                  fair::mq::policy::NullptrInitializer,   // input initialization
-                                  fair::mq::policy::RawPtrDeleter>;       // input deleter
+#include <memory>
+#include <string>
 
 namespace bpo = boost::program_options;
+using namespace std;
```

Comment:
Could we please avoid `using namespace foo;`?

See: d1e148b39ad13331ef741f833a3ea56018ace5f5

(clang-tidy doesn't complain, because it isn't building any Mbs stuff.)

Reply:
Sure.